src/hooks/useTheme.test.ts

```diff
@@ -106,38 +111,48 @@ describe("useTheme", () => {
   ])(
     "handles showSidebarLogo policy (policy: $policyValue, theme: $themeValue, expected: $expectedValue)",
     async ({ policyValue, themeValue, expectedValue }) => {
-      jest.mocked(useAsyncExternalStore).mockReturnValue({
-        data: { ...customTheme, showSidebarLogo: themeValue },
-        isLoading: false,
-      } as AsyncState);
-      jest.mocked(useManagedStorageState).mockReturnValue({
-        data: { showSidebarLogo: policyValue },
-        isLoading: false,
+      jest.mocked(themeStorage.get).mockResolvedValue({
+        ...customTheme,
+        showSidebarLogo: themeValue,
+        lastFetched: Date.now(),
       });
 
-      const { result } = renderHook(() => useTheme());
+      jest.mocked(readManagedStorage).mockResolvedValue({
+        showSidebarLogo: policyValue,
+      });
+
+      const { result, waitForNextUpdate } = renderHook(() => useTheme());
+
+      await waitForNextUpdate();
 
       expect(result.current.activeTheme).toMatchObject({
         ...customTheme,
+        lastFetched: expect.any(Number),
```

Comment:
Referencing https://github.com/pixiebrix/pixiebrix-extension/pull/9228/files/0b71176288772a53625799f76b28655c396c7999..ce8a6ac8037b437d1a40e57ab61e3f18d7f04bb9#r1790227283

```suggestion
        lastFetched,
```